### ISSUE TITLE
Versioning of externs should correspond to the versioning of the library

### ISSUE THREAD

**Issue by Dimous** [foreignsasquatch/raylib-hx#64]
Hello!
So, I tried creating a texture from an SVG and got an error -- `./src/Main.cpp(110): error C3861: 'LoadImageSvg': identifier not found`.
That's because raylib submodule isn't locked to a specific version and raysan5 did this -- https://github.com/raysan5/raylib/commit/d29eb34cfb2688ca11d1a84883c849fa0500881d  a week ago.

I guess, it is not critical, so marking `**@:deprecated** @:native('LoadImageSvg') static function loadImageSvg(fileNameOrString:cpp.ConstCharStar, width:Int, height:Int):RayImage;` would be enouth.

**Comment by MAJigsaw77** [comment]
Did you update the submodules from the lib?

**Comment by Dimous** [comment]
Yes, I had to do it because of an issue with gltf animation. Regardless, versioning makes sense.

**Comment by MAJigsaw77** [comment]
> Yes, I had to do it because of an issue with gltf animation. Regardless, versioning makes sense.

actually, the gltf issue wasn't there, the example you made on that pr was faulty

**Comment by MAJigsaw77** [comment]
Maybe take a look at the current one and keep raylib's commit the same

**Comment by Dimous** [comment]
But updating raylib helped :) Ok ok.

**Comment by Dimous** [comment]
By the way, you are doing a great job with externs, how about JoltPhysics?

**Comment by MAJigsaw77** [comment]
> By the way, you are doing a great job with externs, how about JoltPhysics?

What?

**Comment by Dimous** [comment]
I meant -- you are good at writing externs. If you suddenly start wondering about what other library to write extern for, there is JoltPhysics, just a suggestion.

**Comment by MAJigsaw77** [comment]
> I meant -- you are good at writing externs. If you suddenly start wondering about what other library to write extern for, there is JoltPhysics, just a suggestion.

I understood dude, i just don't know what that library you mentioned is
